packages/l/linphone-desktop/package.yml

```diff
@@ -0,0 +1,56 @@
+name       : linphone-desktop
+version    : 5.2.4
+release    : 1
+source     :
+    - git|https://gitlab.linphone.org/BC/public/linphone-desktop.git : d69e033508265f756fb47b39b76b7d589be159c0
+homepage   : https://linphone.org/
+license    : GPL-3.0-or-later
+component  : network.im
+summary    : A free VoIP and video softphone based on the SIP protocol
+description: |
+    A free VoIP and video softphone based on the SIP protocol
+builddeps  :
+    - pkgconfig(Qt5Multimedia)
+    - pkgconfig(Qt5QuickControls2)
+    - pkgconfig(Qt5Svg)
+    - pkgconfig(Qt5TextToSpeech)
+    - pkgconfig(Qt5UiTools)
+    - pkgconfig(glew)
+    - pkgconfig(libpulse)
+    - pkgconfig(libv4l2)
+    - pkgconfig(x11)
+    - coreutils
+    - doxygen
+    - git
+    - libxcrypt-compat
+    - nasm
+    - qt5-base
+    - qt5-graphicaleffects
+    - qt5-quickcontrols2
+    - yasm
+rundeps    :
+    - glew
+    - qt5-graphicaleffects
+    - qt5-multimedia
+    - qt5-quickcontrols2
+    - qt5-speech
+networking : yes
+setup      : |
+    pip3 install --user pystache six
+    mkdir -p build build/OUTPUT && cd build
+    %cmake .. -DCMAKE_BUILD_PARALLEL_LEVEL=$(nproc) -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Comment:
Pretty sure our CMake macro does the build directory setup for you. I don't think either of those options need to be set, either; the macro will do it for you.

If you can, try using just `%cmake_ninja`.